utils.py

```diff
@@ -82,4 +86,26 @@ def __process_sam_response(sam_response):
 def read_json_file(file_name):
     with open(file_name) as json_file:
         data = json.load(json_file)
-        return data
\ No newline at end of file
+        return data
+    
+def remove_inline_scripts(html_doc):
+    soup = BeautifulSoup(html_doc, 'html.parser')
+
+    # remove math jax script from dom
+    # this would be done together with the for loop below, but semantically it makes sense to separate them
+    # all previews will have this first script, as it is part of the nbconvert lib we use
+    # on the other hand, any other scripts to remove are a result of python libs in a specific notebook relying on javascript to render output 
+    mathJaxUnsafeScript = soup.select_one('script[type="text/x-mathjax-config"]')
+    mathJaxUnsafeScript.decompose()
```

Comment:
Do we want to guard with `if mathJaxUnsafeScript is not None:` or is math jax guaranteed to always be there?

Reply:
Yeah. I thought about this case but it would only occur if we bumped nbconvert to a version that changed its internal implementation, and it would be caught with a the smoke test we have. Doesn't hurt to check though